hphp/system/php/stringish_cast.php

```diff
@@ -0,0 +1,14 @@
+<?hh // strict
```

Comment:
compare to other .php files in system/php using namespaces:

- all have .ns.php extension
- all have `namespace Foo {}` syntax instead of `namespace Foo;`

They're concatenated in a rather straightforward way at present, which depends on these conventions

Reply:
I was not aware of this convention. Should be fixed in a jiffy.